packages/lambda-send-payment/send-email.js

```diff
@@ -127,15 +126,9 @@ ${displayItems(lineItems, '£', '\r\n')}
 
 Total value: £${amount}
 
-The address you submitted was:
-
-${
-  [(`${firstName} ${lastName}`).trim(), address1, address2, address3, postCode].filter(Boolean).join('\r\n')
-}
-
 Your order will be submitted to the factory at the end of the month.
```

Comment:
is this true (especially during lockdown) - do we want to commit to this every month?

Reply:
The idea was to do monthly orders though CV19 has put a spanner in the works there somewhat.  Since production takes 6 weeks it could be a bit of a headache having overlapping orders so we could probably simplify & reword.

A reasonable timeframe may be collect orders for two weeks then six week production, so we place orders every two months.

Reply:
cool. changing to say we aim for two months. will see how it goes.